CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.15.0
+
+This release deprecates `@sentry/hub` and all of it's exports. All of the `@sentry/hub` exports have moved to `@sentry/core`. `@sentry/hub` will be removed in the next major release.
+
+- feat(ember): Add ember-engine-router support (#5905)
+- feat(nextjs): Enable `autoInstrumentServerFunctions` per default (#5919)
+- feat(tracing): Make BrowserTracing heartbeat interval configurable (#5867)
+- fix(remix): Rework dynamic imports of `react-router-dom` (#5897)
+- fix(node): Remove Cookie header from requestdata.headers if cookies should not be sent to Sentry (#5898)
+- fix(utils): accept DSN URLs with empty password (#5902)
```

Comment:
```suggestion
- fix(utils): Accept DSN URLs with empty password (#5902)
```

Not sure. Not be consistent with commit msg.

Reply:
Yeah the changelog can differ slightly from the commit - we can go for readability here.